Tests/DataCollector/DoctrineDataCollectorTest.php

```diff
@@ -49,6 +49,32 @@ public function testCollectEntities()
         $this->assertCount(2, $entities['default']);
     }
 
+    public function testDoNotCollectEntities()
+    {
+        $manager   = $this->getMockBuilder('Doctrine\ORM\EntityManager')->disableOriginalConstructor()->getMock();
+        $config    = $this->getMockBuilder('Doctrine\ORM\Configuration')->getMock();
+        $factory   = $this->getMockBuilder('Doctrine\Common\Persistence\Mapping\AbstractClassMetadataFactory')
+            ->setMethods(['getLoadedMetadata'])->getMockForAbstractClass();
+        $collector = $this->createCollector(['default' => $manager], false);
+
+        $manager->expects($this->never())
+            ->method('getMetadataFactory');
+        $manager->expects($this->any())
```

Comment:
IIRC `->expects($this->any())` can be removed, might depend on the version of PHPUnit

Reply:
it seems to be a required parameter. `getConfiguration` is called outside of the if I just added, so I must give something.

Reply:
I was not talking about the parameter `$this->any()`, but about the entire call `->expects($this->any())`, what about you?

Reply:
So 
```suggestion
        $manager
```

Reply:
done, thanks